templates/zerver/emails/notify_new_login.source.html

```diff
@@ -9,8 +9,8 @@
 <p>This is a notification that a new login to your Zulip account has just occurred.</p>
 <p><b>Login details:</b></p>
 <blockquote>
-    <p>Server: {{ realm_uri }}</p>
-    <p>Account: {{ user.email }}</p>
+    <p>Server: <a href="{{ realm_uri }}" target="_blank">{{ realm_uri }}</a></p>
+    <p>Account: <a href="mailto:{{ user.email }}" target="_blank">{{ user.email }}</a></p>
```

Comment:
We should probably make this part be somehow styled as text if we can; this will always be your own email address, so it's stupid for it to be a link at all.  Not sure if it's possible to do that, but it's worth a try.